code/modules/projectiles/projectile/magic.dm

```diff
@@ -21,6 +18,12 @@
 	damage_type = BRUTE
 	nodamage = 0
 
+/obj/item/projectile/magic/death/on_hit(var/mob/living/carbon/G)
+	. = ..()
+	if(isliving(G))
+		G.adjustBruteLoss(3000)
```

Comment:
This should stay burn.

Wand of Death is meant to be a cheap kill, but something that's easy to correct for with a mere defibbing and they're back on their feet. Unfortunately, due to how IPCs work, this isn't possible, so we went with FireLoss instead, as it's the closest thing.

This will break every bone in their body, quite literally, and probably induce internal bleeding; it'll take about 15 minutes just to get a single one of these victims back on their feet.